banners/mobile/C24_WMDE_Mobile_DE_08/content/BannerSlides.vue

```diff
@@ -24,7 +24,7 @@
 
 	<KeenSliderSlide :is-current="currentSlide === 3">
 		<p>
-			Die durchschnittliche Spende beträgt {{ averageDonation }}, doch bereits 5&nbsp;€ helfen uns weiter. Hat Wikipedia
+			Die durchschnittliche Spende beträgt 22,49&nbsp;€, doch bereits 10&nbsp;€ helfen uns weiter. Hat Wikipedia
```

Comment:
you should use new {{ averageDonation }} variable for 22,49€ here,
Kai doesn't have the variable in his ticket templates yet